apis/python/src/tiledbsoma/_util.py

```diff
@@ -260,3 +261,12 @@ def ms_to_datetime(millis: int) -> datetime.datetime:
     secs, millis = divmod(millis, 1000)
     dt = datetime.datetime.fromtimestamp(secs, tz=datetime.timezone.utc)
     return dt.replace(microsecond=millis * 1000)
+
+
+def validate_platform_config(platform_config: Any) -> None:
+    """Offers a run-time type check on this ingestion/creation argument, which is easy to miskey."""
+    if platform_config is not None:
+        if isinstance(platform_config, TileDBCreateOptions):
```

Comment:
At runtime, `TileDBCreateOptions` is just `dict`. You probably want to check for the presence of certain `TileDBCreateOptions` keys at the root level.

Reply:
@thetorpedodog `TileDBCreateOptions` is a `class`:
https://github.com/single-cell-data/TileDB-SOMA/blob/1.2.3/apis/python/src/tiledbsoma/options/_tiledb_create_options.py#L44

`PlatformConfig` is a `dict`:
https://github.com/single-cell-data/SOMA/blob/python-v1.0.0/python-spec/src/somacore/options.py#L93

Reply:
Whoops, I was confusing it with one of the other types we have. I don’t entirely follow what the intent is here, because a user shouldn’t be creating a CreateOptions themselves in the normal course of business (it’s not on any publicly exposed APIs).